packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -163,6 +166,10 @@ const AppModals = ({
         ComponentToLoad = <NeedRealAccountForCashierModal />;
     }
 
+    if (is_from_eu) {
```

Comment:
lets add a TODO for changing this condition once cashier changes are done.

For testing , lets add is_from_eu and identity and document status are none. ( POI , POA not submitted)